client/src/extension.ts

```diff
@@ -144,7 +106,12 @@ function handleConfigurationChange(event: vscode.ConfigurationChangeEvent) {
         ),
       };
     }
-    configurePlugin();
+    configurePlugin(extensionContext);
+
+    // restart when "deno.path" changes
+    if (event.affectsConfiguration("deno.path")) {
+      vscode.commands.executeCommand("deno.restart");
+    }
```

Comment:
Restarts the language server when someone changes `"deno.path"`. This is a bit more convenient for users.

Side note: These command names and setting names should probably be moved to constants...